search/constants.py

```diff
@@ -154,3 +154,232 @@
     "Video Lectures and Slides": OCW_TYPE_LECTURE_VIDEOS,
     "Video Lectures": OCW_TYPE_LECTURE_VIDEOS,
 }
+
```

Comment:
Moved these constants from `search/indexing_api.py` because that file exceeded 1000 lines (and this seems like a more fitting place anyway).